cfs_config_util/session.py

```diff
@@ -31,11 +31,11 @@
 import os.path
 import os
 
+from csm_api_client.k8s import load_kube_api
```

Comment:
What is still using the version of `AdminSession` defined in this module? Shouldn't we be using the version from `csm_api_client` now?

Reply:
Yes, that's correct. I removed this file.